Source/WTF/wtf/Span.h

```diff
@@ -7,8 +7,52 @@
 //
 //===---------------------------------------------------------------------===//
 
-#ifndef _WTF_LIBCPP_SPAN
-#define _WTF_LIBCPP_SPAN
+#pragma once
+
+#if HAVE(STD_SPAN)
+
+#include <span>
+
+namespace WTF {
+
+inline constexpr std::size_t dynamic_extent = std::numeric_limits<std::size_t>::max();
```

Comment:
Why not this?
```
using std::dynamic_extent;
```

Reply:
Didn't realize that existed.  This line was copied from the `WTF::Span` section.  Will fix.

Reply:
I didn't realize `WTF::dynamic_extent` was being used outside of the `Span.h` header, such as in `Source/WebKit/Platform/IPC/ArgumentCoders.h`:

<https://ews-build.webkit.org/#/builders/77/builds/20081>

Seems like I need this when enabling `std::span`:

`using dynamic_extent = std::dynamic_extent;`

Reply:
Or rather...

`inline constexpr std::size_t dynamic_extent = std::dynamic_extent;`